TCSA.V2/Helpers/ProjectsSubHelpers/AuthProjectsHelper.cs

```diff
@@ -55,7 +55,8 @@ internal static List<Project> GetProjects()
                 {
                     "<a target='blank' href='https://docs.microsoft.com/en-us/aspnet/core/security/authentication/identity?view=aspnetcore-6.0&tabs=visual-studio'>Introduction to Identity</a>",
                     "<a target='blank' href='https://docs.microsoft.com/en-us/aspnet/core/security/authentication/scaffold-identity?view=aspnetcore-6.0&tabs=visual-studio'>Scaffolding Identity into Existing App</a>",
-                    "<a target='blank' href='https://learn.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line'>Logging in C# and .NET</a>"
+                    "<a target='blank' href='https://learn.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line'>Logging in C# and .NET</a>",
+                    "<a target='blank' href='https://learn.microsoft.com/en-us/aspnet/core/security/authentication/identity-api-authorization?view=aspnetcore-8.0'>.NET 8 Web API Identity (for React, Angular, etc.)</a>"
```

Comment:
```suggestion
                    "<a target='blank' href='https://learn.microsoft.com/en-us/aspnet/core/security/authentication/identity-api-authorization'>.NET 8 Web API Identity (for React, Angular, etc.)</a>"
```

if we skip the view type then it alwasy defaults to latest .net version